<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_CheckUpdate</class>
 <widget class="QDialog" name="Dlg_CheckUpdate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>271</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>411</width>
    <height>271</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>411</width>
    <height>271</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New version found</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Current version:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>New version:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="editOld">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="editNew">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://obsproject.com/forum/resources/versus-c-obs-scoreboard.1339/&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; text-decoration: underline; color:#ff007f;&quot;&gt;goto DOWNLOAD&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="releaseNotes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>391</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 230);</string>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
